docs/tutorials/index.md

```diff
@@ -71,17 +70,19 @@ The distribution directory contains `LICENSE` and `NOTICE` files and subdirector
 ## Start up Druid services
 
 Start up Druid services using the `auto` single-machine configuration.
-This configuration includes default settings that are appropriate for this tutorial, such as loading the `druid-multi-stage-query` extension by default so that you can use the MSQ task engine.
+This configuration includes default settings that are appropriate for this tutorial.
+For example, it loads `druid-multi-stage-query` extension so that you can use the MSQ task engine to ingest data using SQL.
 
-You can view that setting and others in the configuration files in the `conf/druid/auto`. 
+You can view the `auto` setting and others in the configuration files in `conf/druid/`. 
```

Comment:
The [`auto` folder](https://github.com/apache/druid/tree/master/examples/conf/druid/auto) has the configuration files
```suggestion
You can view the default settings in the configuration files located in `conf/druid/auto`.
```